bundler/lib/bundler/friendly_errors.rb

```diff
@@ -99,16 +99,16 @@ def serialized_exception_for(e)
       EOS
     end
 
-    def issues_url(exception)
+    def discussions_url(exception)
       message = exception.message.lines.first.tr(":", " ").chomp
       message = message.split("-").first if exception.is_a?(Errno)
       require "cgi"
       "https://github.com/rubygems/rubygems/search?q=" \
-        "#{CGI.escape(message)}&type=Issues"
+        "#{CGI.escape(message)}&type=Discussions"
     end
 
     def new_issue_url
```

Comment:
Given the link now creates a new Discussion, not a new Issue, we should update this function name accordingly.

```suggestion
    def new_discussion_url
```